packages/core/admin/ee/admin/content-manager/pages/EditView/InformationBox/InformationBoxEE.js

```diff
@@ -8,6 +8,7 @@ import {
 import { Field, FieldLabel, FieldError, Flex } from '@strapi/design-system';
 import { useIntl } from 'react-intl';
 import { useMutation } from 'react-query';
+import { get } from 'lodash/fp';
```

Comment:
If I recall correctly we use this import to be tree-shaken by webpack.

```suggestion
import get  from 'lodash/fp/get';
```

Reply:
We don't need lodash here and try to get rid of it wherever possible :) I think the easiest would be if you just revert the FE changes and I'll handle it in a follow-up, because I have to create tests anyways :)